### DISCUSSION TITLE
Simple Time-lapse

### DISCUSSION THREAD

**Discussion by Stumpy023** [gopro/labs#605] [Q&A]
Hello this is my first post so please be sympathetic! I am trying to use a series of GoPros 10/11/12 hero black to record a photo Timelapse. I want it to run from 7am to 7pm everyday. I would like to it as simple and trouble free as possible. I would like the cameras to run on usb, and activate them via switching the usb on and off, I would use simple mains timers for this. What I would like is for the GP to to start shooting on the default camera setting when the power is switched on. This way when I do access the cameras to change the cards I can ensure that they are operation. My attempts so far have failed I am profoundly dyslexic so this may not be helping either.

So far I have tried.

Start at 7:00 or after, finish at 19:00. Shot every 5s. 

<!07:00NmTPp.5!S!43440E!1R

Start at 7:00 or after, finish at 19:00. Shot every 5mins. 

<!07:00NmTPp.300!S!43440E!1R

Start at 7:00 or after, finish at 19:00. Shot every 10minutes. 

<!07:00NmTPp.600!S!43440E!1R

MNAKE=1!MWAKE=1!MBOOT=!SAVEdaily=mTP>07:00<19:00!1S+!19:00E - Not working

!u2N!uE!R power up with battery start default mode.

!MWAKE=2!S - Elegant and simple but only works once.

I am now very confused and frustrated which is not helping me achieve a solution. Anyway help or advise would be much appreciated.

**Comment by dnewman-gpsw** [comment]
If you can, it would be best to use the external timer to manage when the camera is capturing, otherwise you have two timing sources, which will lead to complications (failures.)

So if your USB power can be set to switch on at 7am and off at 7pm, the camera scripting is simpler.

`!MWAKE=2!MBOOT="!Ldaily"!SAVEdaily=mTPp.300!S `

!MWAKE=2 <- this makes the command boot upon USB power,  just like pressing the power button.
!MBOOT="!Ldaily"  <- this is the command to launch on boot.  !Ldaily will load a script from the MISC folder called **daily**
!MSAVEdaily=mTPp.300!S   <- this creates the **daily** script, which is simple as mTPp.300!S  (set TL photo, your interval, Start.)

P.S. Make your your camera's time-lapse preset is set to Photo mode.

**Reply by Stumpy023** [reply]
Thank so much for taking the time to help me I really appreciate your help. I shall try and put this into practice today.
